aggregation/src/Application/Domain/Providers/ProviderA.fs

```diff
@@ -4,6 +4,8 @@ open System
 open NodaTime
 open FsToolkit.ErrorHandling
 
+let providerId = Guid.Parse("05da02cd-65f6-4688-81a2-19053c9dd0b4") |> ProviderId
```

Comment:
Currently, these GUIDs are generated manually and hardcoded. Maybe later we can introduce something that tracks additional information for providers (e.g. the brand name) along with these IDs?